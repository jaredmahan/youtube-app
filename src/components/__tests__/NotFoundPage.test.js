import React from "react";
import { NotFoundPage } from "../NotFoundPage";
import ShallowRenderer from "react-test-renderer/shallow";

describe("<NotFoundPage>", () => {
  it("renders correctly", () => {
    const renderer = new ShallowRenderer();
    renderer.render(<NotFoundPage />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
