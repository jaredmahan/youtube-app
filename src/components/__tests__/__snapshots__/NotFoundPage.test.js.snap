// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFoundPage> renders correctly 1`] = `
<div>
  <h4>
    404 Page Not Found
  </h4>
  <ForwardRef
    to="/"
  >
     Go back to homepage 
  </ForwardRef>
</div>
`;
